### PR TITLE
[zh] Sync contribute  pages for kubernetes-api.md

### PR DESCRIPTION
**Pull request by ydFu**
* Sync with english version in 'Fixed errors in instructions for generating ref docs (#27386)'

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

**Review by zhiguo-lu — COMMENTED** [review]
The document has been synchronized to the latest English version and the revision  is  detailed.

File: content/zh/docs/contribute/generate-ref-docs/kubernetes-api.md [hunk 1]
```diff
@@ -12,24 +12,24 @@ weight: 50
...
+using the [kubernetes-sigs/reference-docs](https://github.com/kubernetes-sigs/reference-docs) generation code.
 
 If you find bugs in the generated documentation, you need to
 [fix them upstream](/docs/contribute/generate-ref-docs/contribute-upstream/).
 
 If you need only to regenerate the reference documentation from the [OpenAPI](https://github.com/OAI/OpenAPI-Specification)
 spec, continue reading this page.
 -->
-本页面展示了如何为 Kubernetes API 更新自动生成的参考文档。
+本页面显示了如何更新为 Kubernetes API 生成的参考文档。
```
[the remark this answers is not in the document]
Comment (ydFu) [reply content/zh/docs/contribute/generate-ref-docs/kubernetes-api.md]: Thank you for suggesting me.

File: content/zh/docs/contribute/generate-ref-docs/kubernetes-api.md [hunk 2]
```diff
@@ -12,24 +12,24 @@ weight: 50
...
 
 Kubernetes API 参考文档是从
 [Kubernetes OpenAPI 规范](https://github.com/kubernetes/kubernetes/blob/master/api/openapi-spec/swagger.json)
-构建的，而工具是从
-[kubernetes-sigs/reference-docs](https://github.com/kubernetes-sigs/reference-docs) 构建的。
+构建的，
+而使用[kubernetes-sigs/reference-docs](https://github.com/kubernetes-sigs/reference-docs) 生成代码。
```
[the remark this answers is not in the document]
Comment (ydFu) [reply content/zh/docs/contribute/generate-ref-docs/kubernetes-api.md]: Thank you for suggesting me.

**Comment by zhiguo-lu** [comment]
thanks
/lgtm

**Comment by ydFu** [comment]
thanks

**Pull request merged**